frontend/src/metabase/css/core/bordered.module.css

```diff
@@ -46,7 +46,8 @@
   border-right: var(--border-size) var(--border-style) var(--color-border);
 }
 
-:global(.border-left) {
+:global(.border-left),
+.borderLeft {
```

Comment:
I'm not sure what this has to do with color class migration. Ideally, I'd expect only `colors.module.css` and any other files composing its color.

Would it be possible to revert this file?

Reply:
That's fine. Let's ignore my comments about border migration then.